benchmarks/modules/cfd/pages/pipestokes/README.adoc

```diff
@@ -174,29 +177,31 @@ The Model CFG (`.cfg`) files allow to pass command line options to {feelpp} appl
 The Cfg file used in this benchmark 
 ----
 directory=tp   <1>
-[fluid] <2>
-filename=$cfgdir/cfd2.json <3>
-mesh.filename=$cfgdir/test.geo <4>
-gmsh.hsize=0.1 <5>
-pc-type=lu #gasm,lu <6>
+case.dimension=2 <2>
+[fluid] <3>
+filename=$cfgdir/pipestokes.json <4>
+mesh.filename=$cfgdir/pipestokes.geo <5>
+gmsh.hsize=0.1 <6>
+pc-type=lu #gasm,lu <7>
 ----
 <1> the directory where the results are exported 
-<2> toolboxe prefix 
-<3> the associated Json file 
-<4> the geometric file 
-<5> the mesh step
-<6> the chosen method for decomposition
+<2>	the dimension of the application, by default 3D
+<3> toolboxe prefix 
+<4> the associated Json file 
+<5> the geometric file 
+<6> the mesh step
+<7> the chosen method for decomposition
 
-We didn' configure the solver, cause in this case our systeme is linear, and by default the solver chosen is the linear one.
+We didn't configure the solver, cause in this case, the systeme is linear, and by default the solver chosen is the linear one.
 
 === Running the model 
 
-The configuration file are in `toolboxes/fluid/pipestokes`.
+The configuration file are in `benchmarks/modules/cfd/examples/pipestokes`.
 Some useful commande lines:
 
 -To executing pipestokes testcase
 ----
-mpirun -np 4 feelpp_toolbox_fluid_2d --config-file /usr/local/share/feelpp/testcases/CFD/pipestokes/pipestokes.cfg
+feelpp_toolbox_fluid --case "github:{owner:feelpp,repo:toolbox,branch:master,path:benchmarks/modules/cfd/examples/pipestokes}" 
```

Comment:
you don't  need to specific `owner`, it is the default. neither `branch` 

have you tested the command ?

Reply:
HI 
Yes I tested the command, it works , I prefered to specify the owner and the branch so that users understand better. but I modified it as well as the other remarks.

Thanks